src/config.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const _ = require('lodash');
+
+const config = {
+    base_url: '',
+    path: '',
+    client_id: '',
+    client_secret: '',
+    configure: configure
+};
+
+module.exports = config;
+
+function configure (new_config) {
+    _.merge(config, new_config);
+
+    if (config.client_expiry === null) {
+        config.client_expiry = 300;
+    }
+
+    if (config.server_expiry === null) {
+        config.server_expiry = 300;
```

Comment:
Why do we need to do this, just set this as default? And if the client wants to override it, we should allow it.